test/e2e/data/eks/cluster-template-eks-ipv6-cluster.yaml

```diff
@@ -45,9 +45,6 @@ spec:
     - name: "vpc-cni"
       version: "${VPC_ADDON_VERSION}"
       conflictResolution: "overwrite"
-    - name: "coredns"
```

Comment:
Do we need to remove this from all EKS tests?

Reply:
Only for cluster templates that are just the control plane. Which is the case here.